src/Microsoft.Extensions.Options/OptionsFactory.cs

```diff
@@ -12,14 +12,17 @@ namespace Microsoft.Extensions.Options
     public class OptionsFactory<TOptions> : IOptionsFactory<TOptions> where TOptions : class, new()
     {
         private readonly IEnumerable<IConfigureOptions<TOptions>> _setups;
+        private readonly IEnumerable<IInitializeOptions<TOptions>> _initializers;
 
         /// <summary>
         /// Initializes a new instance with the specified options configurations.
         /// </summary>
         /// <param name="setups">The configuration actions to run.</param>
-        public OptionsFactory(IEnumerable<IConfigureOptions<TOptions>> setups)
```

Comment:
Should we keep the old ctor for compat, and have it chain to the new ctor? DI would only call the new ctor anyway, but it would avoid a breaking change.